src/storage/lock_manager/client.rs

```diff
@@ -14,6 +14,20 @@ pub type Callback = Box<dyn Fn(DeadlockResponse)>;
 const CQ_COUNT: usize = 1;
 const CLIENT_PREFIX: &str = "deadlock";
 
+/// Builds the `Environment` of the client. All clients should use the same instance.
+fn env() -> Arc<Environment> {
```

Comment:
can we use the same client with pd and raft? 

IMO, I don't like using a static env for gRPC.